.gitignore

```diff
@@ -0,0 +1 @@
+.idea/
```

Comment:
I prefer using global gitignore for things like this.
https://help.github.com/articles/ignoring-files/#create-a-global-gitignore